doc/conf.py

```diff
@@ -15,6 +15,9 @@
 from __future__ import print_function
 import sys
 import os
+
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
Let me elaborate this: I have an old version of scikit-learn (something like 0.14), which ships very old version of `six`. Because project dir is not in the path (at least by the moment `doc/sphinxext/gen_rst.py` is called), the line below uses system's sklearn. This doesn't look right to me.